packages/forma-36-react-components/src/components/Tag/EntityStatusTag/EntityStatusTag.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+
+import EntityStatusTag from './EntityStatusTag';
+
+storiesOf('Components|Tag/EntityStatusTag', module)
+  .addParameters({
+    propTypes: EntityStatusTag['__docgenInfo'],
+  })
+  .add('default', () => (
+    <EntityStatusTag
+      statusLabel={select(
+        'statusLabel',
+        {
+          'Published (default)': 'published',
+          Archived: 'archived',
```

Comment:
```suggestion
          archived: 'archived',
```

Reply:
@mshaaban0 ahh ja fantastisch wunderbar.. I like the entityStatusType.. will change ... :)

Reply:
Would adding the `entityStatusType` prop maybe limit the reuse of the Tag component to only appear in the context of some entities that have a content lifecycle? Could I still use the component for non-content entities (e.g. showing the status of a user invite)?

Reply:
@artibella you should be able to use the Tag `tagType` prop as it is now `<Tag tagType='warning' />`. the entityStatusType is just an extra prop.

Reply:
@artibella I added this as an extra prop so it is configurable like mo described it

Reply:
@mshaaban0 @wichniowski Thanks for clarifying!